### DISCUSSION TITLE
Can we close the func of "BucketNotification" manually?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#10561] [Q&A]
Because of we modify the default user of the bucket, the old user can not access the bucket yet.
![image](https://user-images.githubusercontent.com/13997135/177707041-afaee89f-4c3f-43ff-a7d9-9a6ef3bd43f7.png)

**Comment by thotz** [comment]
Why the owner of the bucket got changed. Like to manage it as internal user, bucket notification relies on that value and fetch it from ob

**Reply by zhucan** [reply]
Because we don't want to use the generated user for the bucket.
